src/pages/index.js

```diff
@@ -1,19 +1,23 @@
 import "../pages/index.css";
 import {
-  initialCards,
   validationSettings,
   editProfileButton,
   addNewCardButton,
   cardForm,
   formElements,
+  editAvatarButton,
+  yesConfirmationButton,
 } from "../utils/constants.js";
 
+import Api from "../components/Api.js";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
```

Comment:
Please, delete `this.disableSubmitButton();` from line 70 in `FormValidator`

You need to **disable** the button only in a `then` block after a **successful response** to be able to click it again if there is a server error.